### PR TITLE
Updating Upworthy and Intevity (twin technologies)

### PR DESCRIPTION
**Pull request by a-thread**
This is a modified version of the [Contributing Guidelines](https://github.com/remoteintech/remote-jobs/blob/master/CONTRIBUTING.md).

This pull request adheres to the repository's [Code of Conduct](https://github.com/remoteintech/remote-jobs/blob/master/CODE_OF_CONDUCT.md).

- [ ] I am an employee of the company mentioned and confirm all included details are correct
- [ ] This PR contains housekeeping only (URL edits, copy changes etc)
- [x] You know your alphabet - company is listed in alphabetical order in the README
- [x] The company directly hires employees. No bootcamps / freelance sites / etc
- [x] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such
- [x] A [company profile](https://github.com/remoteintech/remote-jobs/blob/master/company-profiles/example.md) is included - __Required__ for new additions. (This can be a basic outline but at least something please)
- [x] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
- [x] __Region__ details the geographic regions in which this company's employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
- [x] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

### CODE REVIEW

**Comment by auto-nylen** [comment]
Hi @a-thread, thanks for your contribution!

Our automated checks detected some problems with this pull request (as of commit 9227aaf).  See below for details:

```
upworthy.md: Profile looks complete, but the main readme contains a warning emoji.

1 problem detected
```

Please fix these issues and push a new commit to this pull request.

**Comment by a-thread** [comment]
Removed warning emoji!

**Comment by auto-nylen** [comment]
Hi @a-thread, thanks for updating the pull request!  According to our automated checks, everything looks OK now (as of commit 85c1c83). :heavy_check_mark:

**Comment by auto-nylen** [comment]
Hi @a-thread, thanks for your contribution!

Our automated checks detected some problems with this pull request (as of commit eff883d).  See below for details:

```
README.md: Company is listed out of order: "Intevity" (should be before "Tuft & Needle")

1 problem detected
```

Please fix these issues and push a new commit to this pull request.

**Comment by auto-nylen** [comment]
Hi @a-thread, thanks for updating the pull request!  According to our automated checks, everything looks OK now (as of commit db73df1). :heavy_check_mark:

**Review by dougaitken — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -252,6 +252,7 @@ Name | Website | Region
 [Intercom](/company-profiles/intercom.md) | https://www.intercom.io/ | Worldwide
 [Interpersonal Frequency (I.F.)](/company-profiles/interpersonal-frequency-i-f.md) ⚠️️ | https://ifsight.com/ |
 [Interview Schedule](/company-profiles/interview-schedule.md) | https://interviewschedule.com/ | USA
+[Intevity](/company-profiles/intevity.md) ️ | hhttps://www.intevity.com/ | USA
```
[the remark this answers is not in the document]
Comment (a-thread) [reply README.md]: All set now

**Review by dougaitken — APPROVED** [review]

**Comment by dougaitken** [comment]
Hey @a-thread - can you take a look at the merge conflict, please?

**Comment by a-thread** [comment]
All set!

**Review by dougaitken — APPROVED** [review]

**Comment by lostnco1** [comment]
@a-thread hey im looking at this now and thank you all for your patience. What can i do that will be most effective

**Pull request merged**